src/Akka.Persistence.RavenDB/Journal/RavenDbJournal.cs

```diff
@@ -175,5 +283,59 @@ protected override async Task DeleteMessagesToAsync(string persistenceId, long t
         }
 
         private AsyncReaderWriterLock GetLocker(string persistenceId) => _lockPerActor.GetOrAdd(persistenceId, new AsyncReaderWriterLock());
+
+        public static async Task<Status> EnsureIndexesCreated(RavenDbStore store)
```

Comment:
clean up this method please

Reply:
also move this to `RavenDbStore` and remove the `static`

Reply:
done